app/javascript/src/accessible-autocomplete.js

```diff
@@ -0,0 +1,42 @@
+import accessibleAutocomplete from 'accessible-autocomplete'
```

Comment:
Componentization improvement: These functions would only work one accessible component per page, that works for well for our current page layout, could this be made better to by supporting multiple autocompletes on single page

Reply:
@naseberry Do we need this? I think we'll only ever have one on a page. If so, I'm not sure exactly what you mean or how to achieve it